samples/test/v3/translate_batch_translate_text_with_glossary.test.js

```diff
@@ -71,7 +71,7 @@ describe(REGION_TAG, () => {
     await operation.promise();
   });
 
-  it('should batch translate the input text with a glossary', async () => {
+  it.skip('should batch translate the input text with a glossary [FLAKY]', async () => {
```

Comment:
Just an idea. It could turn out useful to leave a code comment that links to the issue https://github.com/googleapis/nodejs-translate/issues/364. That way, if the future fixer comes straight from the code, as opposed to finding the issue first, we won’t have a stale issue hanging around.

Reply:
:+1: good idea.